src/Components/CreateImageWizard/ValidatedInput.tsx

```diff
@@ -102,6 +52,7 @@ export const HookValidatedInput = ({
   type = 'text',
   inputType,
   warning = undefined,
+  passwordField = false,
```

Comment:
Oh, I can't add a comment to the specific line - on line 60 there is the `const validate`, you should add the `'default'` there instead of in the `useValidation`.

The `validated` is switching between the validation states - meaning the 'default', 'error' or 'success' apply directly to the text input and render the appropriate version of the input.

In `useValidation` the `StepValidation` schema is following:
```typescript
export type StepValidation = {
  errors: {
    [key: string]: string;
  };
  disabledNext: boolean;
};
```
meaning the `error` should contain an error text that will get rendered for an error state. It doesn't affect the formatting of the input and should be empty when there is no problem with validation.

Reply:
ok, so I think I got your point, added another condition in the `validate` function instead of in the `useValidation `